src/app/router/Welcome/Welcome.tsx

```diff
@@ -13,26 +14,36 @@ import ChooseConnector from "@screens/connectors/ChooseConnector";
 import TestConnection from "@screens/Onboard/TestConnection";
 import LocaleSwitcher from "@components/LocaleSwitcher/LocaleSwitcher";
 
+const namespace = { ns: "welcome" };
```

Comment:
This could be any namespace. Maybe something like `namespaceI18n` works?

Reply:
Good idea, fixed!